padertorch/ops/losses/regression.py

```diff
@@ -3,26 +3,53 @@
 from torch.nn import functional as F
 
 
-def _get_scaling_factor(targets, estimates):
+def _get_scaling_factor(target, estimate):
     return torch.unsqueeze(torch.einsum(
-        '...t,...t->...', estimates, targets
-    ), -1) / torch.norm(targets, dim=-1, keepdim=True) ** 2
+        '...t,...t->...', estimate, target
+    ), -1) / torch.norm(target, dim=-1, keepdim=True) ** 2
 
 
-def _reduce(array, reduce):
-    if reduce is None or reduce == 'none':
+def _reduce(array, reduction):
+    if reduction is None or reduction == 'none':
         return array
-    if reduce == 'sum':
+    if reduction == 'sum':
         return torch.sum(array)
-    elif reduce == 'mean':
+    elif reduction == 'mean':
         return torch.mean(array)
     else:
-        raise ValueError(f'Unknown reduce: {reduce}. Choose from "sum", '
-                         f'"mean".')
+        raise ValueError(
+            f'Unknown reduction: {reduction}. Choose from "sum", "mean".')
+
+
+def mse_loss(estimate: torch.Tensor, target: torch.Tensor,
+             reduction: str = 'sum'):
+    """
+    Computes the mse loss.
+    The `reduction` only affects the speaker dimension; the time dimension is always
+    reduced by a mean operation as in [1].
+
+    Args:
+        estimate (... x T): The estimated signal
+        target (... x T, same as estimate): The target signal
+        reduction: 'mean', 'sum' or 'none'/None for batch dimensions
+
+    Returns:
+
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> mse_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(0.9208)
+    >>> mse_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([0.0000, 0.9208])
+    """
+    return _reduce(
+        F.mse_loss(estimate, target, reduction='none').mean(dim=-1).log10(),
```

Comment:
`log10` shouldn't be there for `mse_loss`

Reply:
Yes, fixed